driver/compile.go

```diff
@@ -21,6 +21,13 @@ import (
 	"go.uber.org/zap"
 )
 
+// Global variable for driver package
+// which determines whether this go process will
+// (1) implement parallelism by engaging multiple
+// remote zqd /worker processes, or
+// (2) implement parallelism with local goroutines
```

Comment:
Make this a proper doc comment.
```suggestion
// WorkerURLs, if not empty, causes this process to
// implement parallelism using worker processes
// instead of goroutines.
```